### PR TITLE
Fix basic offline support in Reader

### PR DESCRIPTION
**Pull request by leandroalonso**
This PR fixes the basic offline suport in the new Reader detail.

### To test

1. Open the Reader
2. Save a few posts 
3. Go offline
4. Open the saved posts, they should show the cached images

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift [hunk 1]
```diff
@@ -21,8 +21,8 @@ struct ReaderCSS {
     ///
```
Comment (emilylaguna) [inline WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift:21]: The comments reference a 2 day update period but we've changed to 5. What about changing to reference the `expirationDays` variable?

**Comment by leandroalonso** [comment]
@emilylaguna ready for another look.

I implemented unit tests for 3 different scenarios (each of 'em when online and offline). I guess the logic is now correct.

File: WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift [hunk 2]
```diff
@@ -6,23 +6,25 @@ import Foundation
...
 
     private var expirationDaysInSeconds: Int {
         return expirationDays * 60 * 60 * 24
     }
 
-    private let updatedKey = "ReaderCSSLastUpdated"
+    static let updatedKey = "ReaderCSSLastUpdated"
 
     /// Returns the Reader CSS appending a timestamp
     /// We force it to update every 2 days
```
Comment (emilylaguna) [inline WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift:22]:
```suggestion
    /// We force it to update based on the `expirationDays` property
```

**Comment by leandroalonso** [comment]
@emilylaguna done!

**Pull request merged**